ffi/build.py

```diff
@@ -188,17 +188,21 @@ def main_posix(kind, library_ext):
     else:
         (version, _) = out.split('.', 1)
         version = int(version)
-        if version == 16:
-            msg = ("Building with LLVM 16; note that LLVM 16 support is "
-                   "presently experimental")
+        if version == 20:
+            pass
+        elif version > 20:
+            msg = ("WARNING: Building llvmlite with LLVM version > 20 ({!r}) "
+                   "is untested and may result in problems. Be sure to set "
+                   "LLVM_CONFIG to the right executable path.\nRead the "
+                   "documentation at http://llvmlite.pydata.org/ for more "
+                   "information about building llvmlite.\n".format(version))
             show_warning(msg)
-        elif version != 15:
-
-            msg = ("Building llvmlite requires LLVM 15, got "
-                   "{!r}. Be sure to set LLVM_CONFIG to the right executable "
+        else:
```

Comment:
Corresponding change to the above:
```suggestion
        elif version < 20:
```